### DISCUSSION TITLE
release中的exe无法正常使用

### DISCUSSION THREAD

**Discussion by ride-donkey** [1715173329/UnblockNeteaseMusic#497] [Q&A]
exe程序点开后闪一下就自动关闭

**Comment by pan93412** [comment]
你是使用 `arm64` 還是 `x64` 版本呢？

**Comment by ride-donkey** [comment]
x64
并且该问题发生不稳定,有的时候就会有有的时候就没有.甚至有时重启多遍仍无效,但等第二天用电脑的时候可能就没事了

**Comment by 1715173329** [comment]
打开 cmd 窗口，在里面执行 exe 程序，然后观察输出。

**Comment by ride-donkey** [comment]
node:events:368
      throw er; // Unhandled 'error' event
      ^

Error: listen EACCES: permission denied 0.0.0.0:8080
    at Server.setupListenHandle [as _listen2] (node:net:1317:21)
    at listenInCluster (node:net:1382:12)
    at Server.listen (node:net:1469:7)
    at C:\snapshot\server\precompiled\app.js
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1361:8)
    at processTicksAndRejections (node:internal/process/task_queues:83:21) {
  code: 'EACCES',
  errno: -4092,
  syscall: 'listen',
  address: '0.0.0.0',
  port: 8080
}

**Comment by pan93412** [comment]
請使用 `-p <HTTP Port>:<HTTPS Port>` 命令更改成其他 port，或關掉佔用 8080 port 的應用程式。

**Comment by ride-donkey** [comment]
在powershell中使用 netstat -ano命令后发现8080端口并没有被占用,并且在使用-p命令胡乱输入了几个并未占用的端口仍然是相同的报错.再次尝试后可正常运行.
